code/addons/interactions/template/stories/unhandled-errors.stories.ts

```diff
@@ -0,0 +1,20 @@
+import { global as globalThis } from '@storybook/global';
+import { userEvent, within } from '@storybook/test';
+
+export default {
+  component: globalThis.Components.Button,
+  args: {
+    label: 'Button',
+  },
+  parameters: {
+    actions: { argTypesRegex: '^on[A-Z].*' },
+  },
+};
+
+export const Default = {
```

Comment:
do you need to disable chromatic snapshots for this?

Reply:
Once chromatic supports unhandled errors, we need to disable snapshots yes (and also we need disable the test runner for this story)